session.go

```diff
@@ -115,12 +115,11 @@ func BasicParamsMap(inner SessionHandler) SessionHandler {
 		// the basic information here
 		req.Params["CONTENT_TYPE"] = r.Header.Get("Content-Type")
 		req.Params["CONTENT_LENGTH"] = r.Header.Get("Content-Length")
-		req.Params["HTTPS"] = isHTTPS
```

Comment:
`HTTPS` is a basic CGI variable that needs to be implemented.

From [IBM](https://www.ibm.com/support/knowledgecenter/en/SSVRGU_9.0.1/basic/H_TABLE_OF_CGI_VARIABLES_3561.html):
```
'HTTPS': Indicates if SSL mode is enabled for the server.
```

From [PHP](http://php.net/manual/en/reserved.variables.server.php):
```
'HTTPS': Set to a non-empty value if the script was queried through the HTTPS protocol. 
```